tests/utils/verification/volumeproperties.go

```diff
@@ -90,17 +95,54 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// VerifyAttachedStatus - verify volume is attached and name of the VM attached
+// is consistent on both docker host and ESX
+func VerifyAttachedStatus(name, hostName, esxName string) bool {
+	log.Printf("Confirming attached status for volume [%s]\n", name)
+
+	vmAttachedHost := GetVMAttachedToVolUsingDockerCli(name, hostName)
+	vmAttachedESX := GetVMAttachedToVolUsingAdminCli(name, esxName)
+
+	return (vmAttachedHost == vmAttachedESX)
```

Comment:
this is kind of alarming ... need to add one more check against our expectation. `expectedAttachStatus == vmAttachedHost & vmAttachedESX`